tensorboard/webapp/widgets/data_table/data_table_component.ng.html

```diff
@@ -33,22 +33,33 @@
           <ng-container *ngFor="let header of headers;">
             <td [ngSwitch]="header">
               <span *ngSwitchCase="ColumnHeaders.VALUE_CHANGE">
-                <mat-icon
-                  *ngIf="runData.VALUE_CHANGE >= 0"
-                  class="delta-arrow"
-                  svgIcon="arrow_upward_24px"
-                ></mat-icon>
-                <mat-icon
-                  *ngIf="runData.VALUE_CHANGE < 0"
-                  class="delta-arrow"
-                  svgIcon="arrow_downward_24px"
-                ></mat-icon>
+                <ng-container
+                  *ngTemplateOutlet="arrow; context: {$implicit:runData.VALUE_CHANGE}"
+                ></ng-container>
+                {{ getFormattedDataForColumn(header, runData) }}
+              </span>
+              <span *ngSwitchCase="ColumnHeaders.PERCENTAGE_CHANGE">
+                <ng-container
+                  *ngTemplateOutlet="arrow; context: {$implicit:runData.PERCENTAGE_CHANGE}"
+                ></ng-container>
                 {{ getFormattedDataForColumn(header, runData) }}
               </span>
               <span *ngSwitchDefault
                 >{{ getFormattedDataForColumn(header, runData) }}</span
               >
             </td>
+            <ng-template #arrow let-value>
```

Comment:
This seems like a slightly odd place to locate your template: Can it be moved to the top-level or closer to the top-level of the dom hierarchy in this file? Maybe just move it to the end of the file?

Reply:
I thought it was convenient to have a it close to where it is being used. I moved it to the bottom of the file though.